### PR TITLE
TEC-3475: Add a new scheduled status

### PR DESCRIPTION
**Pull request by mitogh**
Add a new status for "scheduled" or background tasks.

Ticket: [TEC-3475]

[TEC-3475]: https://moderntribe.atlassian.net/browse/TEC-3475

### CODE REVIEW

File: src/Tribe/Aggregator/Tabs/New.php [hunk 1]
```diff
@@ -284,6 +284,14 @@ public function get_result_messages( $queue ) {
 				);
 			}
 
+			$images_scheduled = $queue->activity->get( 'images', 'scheduled' );
+			if ( ! empty( $images_scheduled ) ) {
+				$messages['success'][] = sprintf( // add image import count
```
Comment (tr1b0t) [inline src/Tribe/Aggregator/Tabs/New.php:289]: :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: a2430c8c7a55dfe669c56e4deedd7e92</sub>

File: src/Tribe/Aggregator/Tabs/New.php [hunk 2]
```diff
@@ -284,6 +284,14 @@ public function get_result_messages( $queue ) {
...
+						_n( '%1$d new image was scheduled for import.', '%1$d new images were scheduled for import.', $queue->activity->count( 'images', 'scheduled' ), 'the-events-calendar' ),
```
Comment (tr1b0t) [inline src/Tribe/Aggregator/Tabs/New.php:290]: * :no_entry_sign: Multi-line function call not indented correctly; expected 20 spaces but found 24
* :no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.

<sub>hash: aabf13206ec43ac2dcdc830dbf684559</sub>

File: src/Tribe/Aggregator/Tabs/New.php [hunk 3]
```diff
@@ -284,6 +284,14 @@ public function get_result_messages( $queue ) {
...
+						$queue->activity->count( 'images', 'scheduled' )
```
Comment (tr1b0t) [inline src/Tribe/Aggregator/Tabs/New.php:291]: :no_entry_sign: Multi-line function call not indented correctly; expected 20 spaces but found 24

<sub>hash: a825c365ab2196641bd6cce769d74522</sub>

**Comment by bordoni** [comment]
We just need a changelog here.

**Pull request merged**